packages/@uppy/store-redux/src/index.js

```diff
@@ -1,6 +1,6 @@
 import { nanoid } from 'nanoid/non-secure'
 
-import packageJson from '../package.json'
+import packageJson from '../package.json' with { type: 'json' }
```

Comment:
this is not valid in browsers, right? but it does get transpiled so it should be ok yes?

Reply:
ok let's do it

Reply:
just FYI we removed this plugin in #5834